### PR TITLE
add portconfig.adoc

### PR DESCRIPTION
**Pull request by alfonsosiciliano**
Add entry for PortConfig (portconfig.adoc)

### CODE REVIEW

**Comment by alfonsosiciliano** [comment]
CC @bapt

File: 2022q1/PortConfig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+It is possible to configure a port before the building and installation,
+PortConfig is an utility for setting the port options via a Text User Interface,
+it and its dependecies (libbsddialog and libncurses) are released completely
+under permissive open source licenses. Currently the dialog4ports TUI utility
+performs this task, unfortunately it depends on non-permissive software.
+
+As each terminal has different properties PortConfig can be custimized via
+environment variables to set up the User Interface, for example: menu size,
+theme, borders, and so on, each feature is documented inside the manual.
```
Comment (pauamma) [inline 2022q1/PortConfig:20]: s/and so on,/and so on;/

File: 2022q1/PortConfig [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Further, if a port has a specific 'pkg-help' file, PortConfig will show a Help
+button to open a "popup" with help information.
+
+FreeBSD provides thousands of ports therefore it is not feasible to test
+PortConfig for each use, please report any problem.
```
Comment (pauamma) [inline 2022q1/PortConfig:25]: s/,/;/

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q1/PortConfig (hunk 2 above)
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: fixed

File: 2022q1/PortConfig (hunk 1 above)
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: fixed

File: 2022q1/PortConfig [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
+
+It is possible to configure a port before the building and installation,
+PortConfig is an utility for setting the port options via a Text User Interface,
+it and its dependecies (libbsddialog and libncurses) are released completely
+under permissive open source licenses. Currently the dialog4ports TUI utility
+performs this task, unfortunately it depends on non-permissive software.
+
+As each terminal has different properties PortConfig can be custimized via
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: fxed

File: 2022q1/PortConfig [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Contact: Baptiste Daroussin (port) <bapt@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
+
+It is possible to configure a port before the building and installation,
+PortConfig is an utility for setting the port options via a Text User Interface,
+it and its dependecies (libbsddialog and libncurses) are released completely
+under permissive open source licenses. Currently the dialog4ports TUI utility
+performs this task, unfortunately it depends on non-permissive software.
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: fixed

File: 2022q1/PortConfig [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Contact: Alfonso Sabato Siciliano (upstream) <asiciliano@FreeBSD.org>  
+Contact: Baptiste Daroussin (port) <bapt@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
+
+It is possible to configure a port before the building and installation,
+PortConfig is an utility for setting the port options via a Text User Interface,
+it and its dependecies (libbsddialog and libncurses) are released completely
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: fixed

File: 2022q1/PortConfig [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+link:https://gitlab.com/alfix/portconfig/[Repository portconfig] URL: link:https://gitlab.com/alfix/portconfig/[https://gitlab.com/alfix/portconfig/] +
+
+Contact: Alfonso Sabato Siciliano (upstream) <asiciliano@FreeBSD.org>  
+Contact: Baptiste Daroussin (port) <bapt@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
+
+It is possible to configure a port before the building and installation,
+PortConfig is an utility for setting the port options via a Text User Interface,
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: OK "."

File: 2022q1/PortConfig [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+Links
+link:https://gitlab.com/alfix/portconfig/[Repository portconfig] URL: link:https://gitlab.com/alfix/portconfig/[https://gitlab.com/alfix/portconfig/] +
+
+Contact: Alfonso Sabato Siciliano (upstream) <asiciliano@FreeBSD.org>  
+Contact: Baptiste Daroussin (port) <bapt@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
+
+It is possible to configure a port before the building and installation,
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: OK "."

File: 2022q1/PortConfig [hunk 8]
```diff
@@ -0,0 +1,28 @@
+== PortConfig
+
+Links
+link:https://gitlab.com/alfix/portconfig/[Repository portconfig] URL: link:https://gitlab.com/alfix/portconfig/[https://gitlab.com/alfix/portconfig/] +
+
+Contact: Alfonso Sabato Siciliano (upstream) <asiciliano@FreeBSD.org>  
+Contact: Baptiste Daroussin (port) <bapt@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to allow a simple way for users and
+administrators to install applications.
```
[the remark this answers is not in the document]
Comment (alfonsosiciliano) [reply 2022q1/PortConfig]: OK

**Comment by alfonsosiciliano** [comment]
Fixed

**Comment by bapt** [comment]
LGTM

**Review by pauamma — APPROVED** [review]

**Review by pauamma — APPROVED** [review]

**Review by pauamma — APPROVED** [review]

**Pull request merged**